pkg/azureclients/diskclient/interface.go

```diff
@@ -26,7 +26,7 @@ import (
 
 const (
 	// APIVersion is the API version for compute.
-	APIVersion = "2019-11-01"
+	APIVersion = "2020-06-30"
```

Comment:
@andyzhangx  do you know whether this API version is available on all clouds?

Reply:
need to confirm with storage team, also for this feature, should test on csi driver first, let's hold this PR now.

Reply:
Already tested with the csi driver. Was able to create ultra disks successfully.

Reply:
@andyzhangx Now that we have the confirmation that this version is available on all the clouds, can we stop holding this PR?

Reply:
@nearora-msft could you port this code change to csi driver vendor first and then add csi driver related change? I would like to confirm it works in CSI driver e2e test first, thanks.

Reply:
Created PR : https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/661